vendor/github.com/vrischmann/go-metrics-influxdb/influxdb.go

```diff
@@ -0,0 +1,219 @@
+package influxdb
+
+import (
+	"fmt"
+	"log"
+	uurl "net/url"
+	"time"
+
+	"github.com/influxdata/influxdb/client"
+	"github.com/rcrowley/go-metrics"
+)
+
+type reporter struct {
+	reg      metrics.Registry
+	interval time.Duration
+
+	url      uurl.URL
+	database string
+	username string
+	password string
+	tags     map[string]string
+
+	client *client.Client
+}
+
+var sendNow = make(chan bool, 1)
+var endSending = make(chan bool, 1)
+
+// SendMetricsNow Sends metrics immediately
+func SendMetricsNow() {
```

Comment:
I add a SendMetricsNow function here, letting metrics send manually, and wait until metrics sent.